packages/mobile/src/account/Account.tsx

```diff
@@ -173,11 +173,11 @@ export class Account extends React.Component<Props, State> {
     } else {
       return (
         <View style={style.devSettings}>
-          <View style={style.devSettingsItem}>
+          {/* <View style={style.devSettingsItem}>
             <TouchableOpacity onPress={this.revokeNumberVerification}>
               <Text>Revoke Number Verification</Text>
             </TouchableOpacity>
-          </View>
+          </View> */}
```

Comment:
Maybe. It's currently broken and we'll need to see how it can be done with the new Attestation contract which collects attestations instead of replacing them when re-verifying.